packages/tasks/src/model-libraries.ts

```diff
@@ -235,12 +235,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/Unbabel/COMET/",
 		countDownloads: `path:"hparams.yaml"`,
 	},
-	contexttab: {
-		prettyLabel: "ConTextTab",
-		repoName: "ConTextTab",
-		repoUrl: "https://github.com/SAP-samples/contexttab",
+	"sap-rpt-1-oss": {
```

Comment:
can you move the entry down in the mapping to keep alphabetical order? (between `sample-factory` and `sapiens`)

Reply:
Perfect, thanks!